emcc.py

```diff
@@ -918,6 +922,24 @@ def detect_fixed_language_mode(args):
       if separate_asm:
         shared.Settings.SEPARATE_ASM = os.path.basename(asm_target)
 
+      if 'EMCC_STRICT' in os.environ:
+        shared.Settings.STRICT = os.environ.get('EMCC_STRICT') != '0'
+
+      STRICT = ([None] + filter(lambda x: x.startswith('STRICT='), settings_changes))[-1]
+      if STRICT:
+        shared.Settings.STRICT = int(STRICT[len('STRICT='):])
```

Comment:
Oh I see, because we need to do the library stuff first.

But then, let's comment that above here and not just below.